src/app/server/Mdns.cpp

```diff
@@ -93,13 +98,15 @@ CHIP_ERROR GetCommissionableInstanceName(char * buffer, size_t bufferLen)
 /// Set MDNS operational advertisement
 CHIP_ERROR AdvertiseOperational()
 {
-    uint64_t fabricId;
+    uint64_t fabricId = 5544332211; // default value
 
+#if CONFIG_DEVICE_LAYER
```

Comment:
The AdvertiseOperational may be too "cast" to devices (and require direct access to underlying storage APIs). It seems like you have found another spot where your usecase is impacted by the current API assumptions. However, rather than raise and discuss how to solve for all cases, there is yet more `#ifdef` here and more assumptions being added.

Reply:
No changes to Mdns.cpp anymore in the latest diff.